cli/src/commands/jobs.rs

```diff
@@ -127,9 +127,6 @@ pub async fn handle_submission(api: &mut PhylumApi, matches: &clap::ArgMatches)
         synch = true;
 
         jobs_project = JobsProject::new(api, matches).await?;
-        if jobs_project.lockfiles.is_empty() {
-            return Err(anyhow!("Missing lockfile parameter"));
-        }
```

Comment:
This code was removed since we already check for this in `config::lockfiles(...)` which is called from `JobsProject::new(...)` and the error is just bubbled up throughout. So it should be unreachable code anyway.